Tests/AEXMLTests.swift

```diff
@@ -36,14 +36,14 @@ class AEXMLTests: XCTestCase {
     // MARK: - Helpers
     
     func URLForResource(fileName: String, withExtension: String) -> NSURL {
-        let bundle = NSBundle(forClass: AEXMLTests.self)
-        return bundle.URLForResource(fileName, withExtension: withExtension)!
+        let bundle = Bundle(for: AEXMLTests.self)
+        return bundle.urlForResource(fileName, withExtension: withExtension)!
     }
     
     func xmlDocumentFromURL(url: NSURL) -> AEXMLDocument {
         var xmlDocument = AEXMLDocument()
         
-        if let data = NSData(contentsOfURL: url) {
+        if let data = NSData(contentsOf: url as URL) {
```

Comment:
`as URL` may be removed if you change the parameter type from `NSURL` to `URL`, which I think Apple encourage us to use.